src/sentry/static/sentry/app/views/organizationEvents/eventsTable.jsx

```diff
@@ -16,6 +16,65 @@ import space from 'app/styles/space';
 
 import EventsContext from './utils/eventsContext';
 
+class EventsTableBody extends React.PureComponent {
+  static propTypes = {
+    events: PropTypes.array,
+    organization: SentryTypes.Organization,
+    utc: PropTypes.bool,
+    projectsMap: PropTypes.object,
+  };
+
+  getEventTitle(event) {
+    const {organization, projectsMap} = this.props;
+    const project = projectsMap.get(event.projectID);
+    const trimmedMessage = event.message.split('\n')[0].substr(0, 100);
+
+    if (!project) {
```

Comment:
In what situation would a project not be found in the map? In this situation wouldn't the rest of the render function fail anyway since many other nodes appear to rely on a valid project being present?

Reply:
Removed